tests/test_morph.py

```diff
@@ -32,18 +87,65 @@ def test_get_1(self):
         npt.assert_almost_equal(expected, actual[:2])
 
     def test_get_2(self):
-        self.nodes.get.return_value = 'small_morph'
-        self.nodes.positions.return_value = pd.Series({
-            'x': 100.0,
-            'y': 200.0,
-            'z': 300.0,
-        })
-        self.nodes.orientations.return_value = -np.identity(3)
-        actual = self.test_obj.get(42, transform=True).points
+        node_id = 0
+        actual = self.test_obj.get(node_id, transform=True).points
+
+        # check that the input node positions / orientation values are still the same
+        pdt.assert_series_equal(self.nodes.positions(node_id),
+                                pd.Series([101., 102., 103.],
+                                          index=[Node.X, Node.Y, Node.Z], name=0))
+        npt.assert_almost_equal(
+            self.nodes.orientations(node_id),
+            [
+                [1,  0., 0.],
+                [0., 0., -1.],
+                [0., 1., 0.],
+            ],
+            decimal=6
+        )
+        assert len(actual) == 32
+        # swc file
+        # index       type         X            Y            Z       radius       parent
+        # 1           1    -0.320000     1.000000     0.000000     0.725000          -1
+        # 2           1    -0.320000     0.900000     0.000000     0.820000           1
+
+        # rotation around the x axis 90 degrees counter clockwise
+        expected = [
```

Comment:
I'm puzzled here. We had a 90 degrees rotation of a point (101,102,103), yet we see that coordinates didn't change almost at all `[100.68, 102, 104.0, 0.725]`. Shouldn't at least one of coordinates become negative? What am I missing?

Reply:
The rotation around the x axis is done before the translation. The translation being positive and far superior to the morph size you will only have positive number. 
Plus, the rotation basically keep X identical and switch the y and z axis for the morph. 
I added comments.

Reply:
That clarifies things. Thank you.